zerver/lib/message.py

```diff
@@ -320,6 +322,7 @@ def render_markdown(message, content, realm=None, realm_alert_words=None, messag
             if user_id in message_user_ids:
                 possible_words.update(set(words))
 
+    available_slash_commands = set(get_slash_commands_by_realm(realm, is_active=True))
```

Comment:
Sorry I am not able to get your point. Do you mean that I should rename `available_slash_commands` to realm_slash_commands` ?